tutorials/01-DataJoint Basics.ipynb

```diff
@@ -468,7 +548,27 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Go ahead and insert a few more mice into your table before moving on."
+    "The error message suggests to include a new argument `skip_duplicates=True` in the `.insert1()` method to skip inserting duplicated entries. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "Mouse.insert1(\n",
+    "{'mouse_id': 0,\n",
+    " 'dob': '2018-01-01',\n",
+    " 'sex': 'M',\n",
+    "}, skip_duplicates=True)"
+   ]
```

Comment:
This should be removed from here. It distracts from making a point about what happens if there is a duplicate.

Reply:
I agree. Thank you for your suggestions, @kushalbakshi !

Reply:
I have incorporated this modification into a new commit.